### ISSUE TITLE
Travis is failing after SQL Server security fixes

### ISSUE THREAD

**Issue by mhujer** [zendframework/zf1#620]
Can you please have a look @weierophinney ?

```

1) Zend_Db_Adapter_Pdo_MysqlTest::testAdapterQuoteNullByteCharacter
Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
-''1\000''
+''1\\000''

```

https://travis-ci.org/zendframework/zf1/jobs/80462389#L483

**Comment by mhujer** [comment]
@weierophinney ping?

**Comment by mhujer** [comment]
@weierophinney ping?

**Comment by froschdesign** [comment]
Fixed with #638
